packages/cli/src/lib/get-oxygen-deployment-data.ts

```diff
@@ -39,6 +45,7 @@ export async function getOxygenDeploymentData({
     }
 
     config.storefront = await linkStorefront(root, session, config, {
+      force: true,
```

Comment:
Follow-up form the previous comment:
Can we start passing here the storefronts we already have instead of requesting them again? -- we'd also need to modify `linkStorefront` to accept an optional `storefronts` param.

```suggestion
      force: true,
      storefronts: await storefrontsPromise,
```